### ISSUE TITLE
Acessando projeto via NGROK com instalação local no Docker Windows -Network Error

### ISSUE THREAD

**Issue by M4rceC4r** [ticketz-oss/ticketz#136]
Acessado localmente após a instalação, ok
Acessando externamente:
Criando um túnel através do ngrok, aparece a tela de login, porém sem a logo na página inicial. Apenas os campos login e senha.
Assim que informando retorna o erro Network Error
Alerta no NODE aparece parcialmente no dashboard do ngrok

**Comment by allgood** [comment]
@M4rceC4r , favor detalhar melhor o seu setup, você tá usando docker com configuração local? ou executando direto? Como estão os backend/.env e o frontend/public/config.json?

**Comment by M4rceC4r** [comment]
Instalação feita via Docker Desktop no Windows 11 com arquivo: .env-frontend-local e .env-backend-local configurados para IP da maquina local 192.168....
O sistema localmente funciona ok.... gerando o túnel no ngrok configurado corretamente, não é possível fazer o login. Seguindo a mensagem Network Error. 
Poderia ser porque o link do túnel gerado pelo ngrok é  em HTTPS e o ticketz esta em HTTP?

**Comment by robertonfaustino** [comment]
Gostaria de saber se conseguiu? também tenho essa duvida

**Comment by allgood** [comment]
@M4rceC4r @robertonfaustino 

para acessar externo é preciso configurar pelos arquivos "acme" ao invés de "local".

**Comment by robertonfaustino** [comment]
> @M4rceC4r @robertonfaustino
> 
> para acessar externo é preciso configurar pelos arquivos "acme" ao invés de "local".

certo, e a url eu colocaria no frontend_host? frontend_url e backend_url deixo como está?
![image](https://github.com/user-attachments/assets/8049b777-8268-4085-ab9d-0cec606265bb)

**Comment by robertonfaustino** [comment]
deu boa, obrigado
